easyrsa3/easyrsa

```diff
@@ -100,8 +101,16 @@ cmd_help() {
 
       This mode uses the <filename_base> as the X509 CN."
 			opts="
-        nopass  - do not encrypt the private key (default is encrypted)
-        inline  - create an inline credentials file for this node" ;;
+        nopass  - do not encrypt the private key (default is encrypted)" ;;
+		build-inline) text="
+  build-inline <filename_base>
+      Create an inline file for node <filename_base>
+
+      The inline file is:
+      * created in your PKI directory: ./inline/<filename_base>.conf
```

Comment:
We should use .ovpn for OpenVPN configuration files - these are automatically recognized on Windows/Mac with Tunnelblick or OpenVPN installed.

Reply:
`.ovpn` will break the GUI.